src/main/java/com/sphy/hotelmanagementapplication/dto/RoomDTO.java

```diff
@@ -24,29 +22,6 @@ public class RoomDTO implements Serializable {
 
 	private boolean disabled;
 
-	public RoomDTO(Long id, String name, Long hotel) {
-		Id = id;
-		this.name = name;
-		this.hotel = hotel;
-	}
-
-	public RoomDTO() {
```

Comment:
If we are to sue the empty constructor, the best practice is to show it in class. Add the empty constructor please.